k-distribution/src/main/scripts/bin/krun

```diff
@@ -612,7 +618,7 @@ if ! $dryRun; then
 
     case "$outputMode" in
       pretty)
-      execute kprint "$kompiledDir" "$kore_output" $color > "$outputFile"
+      execute kprint "$kompiledDir" "$kore_output" $color $filterSubst > "$outputFile"
       ;;
 
       kore)
```

Comment:
This whole block should probably be replaced with just a call to `kore-print`, I would think? So that we don't duplicate the logic of selecting `kast` or `kprint` in two places?

Reply:
I mean, I do intend to refactor the krun script to make better use of these types of things, but I wanted to do that in a separate PR. The changes in this PR are just to make things stylistically consistent and bring krun up to feature parity with kore-print.